rules/what-the-user-experience-should-be-like/rule.md

```diff
@@ -0,0 +1,42 @@
+---
+type: rule
+title: Do you know what the user experience should be like?
+uri: what-the-user-experience-should-be-like
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan/
+created: 2014-03-14T02:39:00.000Z
+archivedreason: outdated
+guid: 24f2a833-2d72-40d7-aeb0-3acea67bfc7f
+---
+As per the rule [Do you know there should be a standard menu item "Check for Updates"?](RulesToBetterWindowsForms.aspx#CheckForUpdates) on Rules to Better Windows UI, you should have a standard menu item "Check for Updates" in the Help menu.
```

Comment:
Hey @fenix2222 

- [ ] 1. Can you update this link? I assume the rule was moved already

Reply:
I've removed this link for @fenix2222 since I couldn't find it